x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -547,6 +547,18 @@ export function App({
     visualizationState: visualization,
   });
 
+  // todo: isLoading is always false
```

Comment:
is this expected ?

Reply:
It shouldn't. I can see in `lens_slice.ts` setting it to `true`.